ios/FluentUI/Tab Bar/SideTabBarTokens.swift

```diff
@@ -0,0 +1,57 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import UIKit
+
+/// Design token set for the `TabBar`.
+open class SideTabBarTokens: ControlTokens {
+    /// The width of the SideTabBar
+    open var sideTabBarWidth: CGFloat { 62.0 }
+
+    /// The safe spacing for the avatar
+    open var avatarViewSafeTopSpacing: CGFloat { 18.0 }
+
+    /// The minimum spacing for the avatar
+    open var avatarViewMinTopSpacing: CGFloat { 36.0 }
+
+    /// The
+    open var avatarViewTopStackViewSpacing: CGFloat { 34.0 }
+
+    /// The safe spacing for the bottom StackView
+    open var bottomStackViewSafeSpacing: CGFloat { 14.0 }
+
+    /// The min spacing for the bottom StackView
+    open var bottomStackViewMinSpacing: CGFloat { 24.0 }
```

Comment:
I know that not all the spacing values are available in GlobalTokens, but do we want to use the global token values when possible?

Reply:
Not all of these values directly map to appropriate global tokens, but I added the ones that made sense.